bindings/java/src/main/java/org/apache/opendal/BlockingOperator.java

```diff
@@ -30,12 +44,12 @@ public void write(String fileName, String content) {
         write(nativeHandle, fileName, content);
     }
 
-    public String read(String s) {
-        return read(nativeHandle, s);
+    public String read(String fileName) {
```

Comment:
Is `fileName` widely used in Java? How about `path`?

Reply:
I don't know. I tend to keep consistent with how OpenDAL names it. Do you think `path` a good idea? If so, let me update all occrances.

Reply:
> Do you think `path` a good idea?

Yes, OpenDAL uses `path` for all public API>

Reply:
Updated.